tests/models/test_semantic_segmentation.py

```diff
@@ -707,7 +714,7 @@ def test_behavior_bcss_local(remote_sample, tmp_path):
     save_dir = pathlib.Path(tmp_path)
 
     _rm_dir(save_dir)
-    wsi_breast = pathlib.Path(remote_sample)
+    wsi_breast = pathlib.Path(remote_sample("wsi4_4k_4k_svs"))
```

Comment:
this is again a 4kx4k image.

Reply:
This test is local test and is skipped on travis so I dont think we need to change anything there.
You can see the
> @pytest.mark.skip(reason="Local manual test, not applicable for travis.")

on top of the function.